TechtonicaModLoader/MyClasses/Managers/ModManager.cs

```diff
@@ -16,6 +17,10 @@ public static class ModManager
     {
         // Objects & Variables
 
+        // The release of 1.0, used to filter mods that are likely broken.
+        internal static readonly DateTime techtonicaReleaseDate = new DateTime(2024, 11, 7);
+        internal static HashSet<string> allowedMods = new HashSet<string> { "BepInExPack", "UnityAudio" };
```

Comment:
Couple more to add to this list:
- Helium
- Official BepInEx ConfigurationManager
- UnityExplorer
- LongStackInserters

Reply:
Added and tested.